### PR TITLE
Allowing stdin in the case of comment -F

### PR DESCRIPTION
**Pull request by hazbo**
As mentioned in #56, passing - for stdin with -F will now read. This is
partially inspired by how git approaches this issue. Here are a couple of
examples of how comments can be made as of this commit:

```
echo 'hello!' | git appraise comment -F - HASH

git appraise comment -F - HASH
(reading comment from standard input)
hello!
```

(ctrl+D to exit from the attached version)
Feedback / thoughts would be appreciated! :)

### CODE REVIEW

File: commands/input/input.go [hunk 1]
```diff
@@ -70,6 +72,38 @@ func LaunchEditor(repo repository.Repo, fileName string) (string, error) {
...
+				return "", fmt.Errorf("Error reading from stdin: %v\n", err)
+			}
+			return string(output), err
+		}
+
+		fmt.Printf("(reading comment from standard input)\n")
+		var output bytes.Buffer
+		s := bufio.NewScanner(os.Stdin)
+		for s.Scan() {
+			output.Write(s.Bytes())
```
Comment (ojarjur) [inline commands/input/input.go]: I think we need to add a newline to what is written in order to compensate for s.Scan() trimming the separator (which defaults to newline).

**Comment by ojarjur** [comment]
Thanks for putting this together.

Overall, I think it looks good. The only feedback I have is the inline comment about `Scan()` dropping newlines that it encounters.

**Comment by hazbo** [comment]
Hey Omar! Sorry this took me so long to get back to. This seems to do the trick.

If this seems okay with you, would it be a good idea to now implement this among the other relevant subcommands?

**Comment by ojarjur** [comment]
> Sorry this took me so long to get back to.

No need to apologize. I really appreciate the amount of work you've put in to this.

> If this seems okay with you, would it be a good idea to now implement this among the other relevant subcommands?

Yes, I think this looks really good now. I'll get this submitted tomorrow unless you want me to wait. The other commands can always go in a later PR.

Thanks again for all of your help.

**Comment by hazbo** [comment]
Alright! Sounds good to me. I'll get the other commands done soon as I can in a separate PR.

**Pull request merged**